### PR TITLE
New stdoutmetric encoder with ignore timestamp

### PR DESCRIPTION
**Pull request by Petrie**
Resolve: https://github.com/open-telemetry/opentelemetry-go/issues/3223

### CODE REVIEW

**Comment by MrAlias** [comment]
The approach showcased in https://github.com/open-telemetry/opentelemetry-go/pull/3529 added an option instead of a dedicated encoder. That approach would apply to all encoders instead of a single one.

Should that approach be taken instead?

**Comment by Petrie** [comment]
> The approach showcased in #3529 added an option instead of a dedicated encoder. That approach would apply to all encoders instead of a single one.
> 
> Should that approach be taken instead?

I apologize for not noticing the "POC" pull request. After reviewing the code structure, I believe it is more organized than mine. I will refactor my code to avoid using a dedicated encoder.

**Review by MrAlias — COMMENTED** [review]
Over all looks good. The testing needs some coverage added to it.

**Comment by MrAlias** [comment]
Please be sure to add a changelog entry.

**Review by pellared — APPROVED** [review]
_I have forgotten to click "submit review" a few hours ago_ 😆

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ This release drops the compatibility guarantee of [Go 1.18].
 
 ### Added
 
+- The `WithoutTimestamps` option to `go.opentelemetry.io/otel/exporters/stdout/stdoutmetric` to sets all timestamps to zero. (#3828)
```
Comment (MrAlias) [inline CHANGELOG.md:30]: This needs to be moved to the unreleased section

**Review by Aneurysm9 — APPROVED** [review]

**Pull request merged**